core/templates/dev/head/css/oppia.css

```diff
@@ -4981,3 +4981,11 @@ md-card.preview-conversation-skin-supplemental-card {
   padding: 0;
   width: 1px;
 }
+
+.ItemSelectionInput md-checkbox .md-icon {
```

Comment:
The naming of this isn't standard css, perhaps `.item-selection-input` would be better.